exporter/containerimage/export.go

```diff
@@ -203,6 +203,7 @@ func (e *imageExporterInstance) Export(ctx context.Context, src exporter.Source)
 	}
 
 	resp["containerimage.digest"] = desc.Digest.String()
+	resp["containerimage.size"] = strconv.FormatInt(desc.Size, 10)
```

Comment:
`size` -> `descriptorsize`? Probably `size` should be used for the total size of the image blobs.

Reply:
We should probably just use `containerimage.descriptor`with the full JSON descriptor. Maybe even deprecate "containerimage.digest" over time.

Reply:
ping @ndeloof

Reply:
> size should be used for the total size of the image blobs.

Which is what I was looking for, seems I went wrong trying to guess the API meaning :P

@tonistiigi I don't have any strong opinion on this.

Reply:
@ndeloof Change it to "descriptor" please so we can merge this. We can do the possible deprecation of "digest" in a follow-up.

Reply:
Better for you if you have a better implementation idea in mind to do this on your own, as this will anywya require subsequent PRs on buildx and dockerd